check_test.go

```diff
@@ -269,7 +297,7 @@ var testcases = map[string]struct {
 				RequireContactMethod: ol.Bool(true),
 				ContactMethod:        &slackType,
 				TeamTagKey:           "updated_at",
-				TeamTagPredicate:     &ol.PredicateInput{
+				TeamTagPredicate: &ol.PredicateInput{
```

Comment:
this feels like an incorrect change - the spacing was removed which is against the go style

Reply:
I'm not sure... goland is doing that automatically when I save it.  Tried moving it back but it keeps removing the whitespace.  Is it because it's setting it to a struct that has fields that are nested/indented on the next line so it just keeps that whole top line together? I haven't customized anything of the go styling so not really sure why it would act differently

Reply:
ok lets ignore for now